hack/bootstrap-ginkgo.sh

```diff
@@ -4,12 +4,16 @@ set -e
 
 source $(dirname "$0")/common.sh
 
-_out/tests/ginkgo version
+FOLDERS="${KUBEVIRT_DIR}/staging/src/kubevirt.io/ ${KUBEVIRT_DIR}/pkg/ ${KUBEVIRT_DIR}/cmd/ ${KUBEVIRT_DIR}/tests/framework/"
```

Comment:
I think you should also add `./tools`
```shell
> find ./tools/ -regex .*_test.go
./tools/util/marshaller_test.go
./tools/cache/cache_suite_test.go
./tools/cache/time-defined-cache_test.go
```


I think that in general we better search in the entire repo, except for specific folders like `./vendor` and `./tests` to ensure we cover everything (even tests that will be added in the future).

Reply:
Okay after a small research, it seems that doing so will be a bit ugly (`find` requires all excluded folders to be provided separately):
```shell
EXCLUDE_DIRS="./vendor ./tests"
find . $(for dir in $EXCLUDE_DIRS; do echo -path $dir -prune -o; done) -name '*_test.go' -print
```

So I'm not sure if it worths making the code this ugly :)
WDYT?

Reply:
So apparently the `/tools` tests don't even compile
```
kubevirt/tools/util/marshaller_test.go:32:195: not enough arguments in call to components.NewHandlerDaemonSet
```

And are not even attempted to run https://github.com/kubevirt/kubevirt/blob/941b04cb9f90745ae2038b13c8bb82eea55a16dd/hack/bazel-test.sh#L7